packages/mos-gateway/src/CoreMosDeviceHandler.ts

```diff
@@ -75,13 +75,15 @@ export class CoreMosDeviceHandler {
 	private _pendingStoryItemChanges: Array<IStoryItemChange> = []
 	private _pendingChangeTimeout: number = 60 * 1000
 	private mosTypes: MosTypes
+	private _openMediaHotStandby: boolean
 
 	private _messageQueue: Queue
 
-	constructor(parent: CoreHandler, mosDevice: IMOSDevice, mosHandler: MosHandler) {
+	constructor(parent: CoreHandler, mosDevice: IMOSDevice, mosHandler: MosHandler, openMediaHotStandby: boolean) {
```

Comment:
I'm wondering if this should be put into an options object? Carrying these opaque flags enabling out-of-spec behavior seems like something that will get problematic very quickly. What about?

```suggestion
	constructor(parent: CoreHandler, mosDevice: IMOSDevice, mosHandler: MosHandler, opts?: { openMediaHotStandby? : boolean }) {
```